README.rst

```diff
@@ -23,7 +23,10 @@ Status
     * - Coverage
       - .. image:: https://codecov.io/gh/brocksam/pyproprop/branch/master/graph/badge.svg
             :target: https://codecov.io/gh/brocksam/pyproprop
-
+    * - conda-forge
```

Comment:
Can we make the "conda-forge" table title read "Anaconda" instead?

Reply:
Addressed in 3195c0c